token_transfer_private.go

```diff
@@ -0,0 +1,39 @@
+package hedera
+
+import (
+	"github.com/hashgraph/hedera-protobufs-go/services"
+)
+
+type _TokenTransfer struct {
+	Transfers        []*HbarTransfer
+	ExpectedDecimals *uint32
+}
+
+func _TokenTransferPrivateFromProtobuf(pb *services.TokenTransferList) *_TokenTransfer {
+	if pb == nil {
+		return &_TokenTransfer{}
+	}
+
+	var decimals *uint32
+	if pb.ExpectedDecimals != nil {
+		temp := pb.ExpectedDecimals.GetValue()
+		decimals = &temp
+	}
+
+	return &_TokenTransfer{
+		Transfers:        _HbarTransferFromProtobuf(pb.Transfers),
+		ExpectedDecimals: decimals,
+	}
+}
+
+func (transfer *_TokenTransfer) _ToProtobuf() []*services.AccountAmount {
```

Comment:
Hmm, I know I did the same thing in Java because I really would like to just keep `[to|from]Protobuf()` names, but perhaps we should just do `[to|from]ProtobufMany()` methods instead?

Reply:
I think this should be fine for now, we can just add `[to|from]ProtobufSingle()` if we need to.